sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -757,7 +757,7 @@ public void testWriteUnknown() throws Exception {
                 .withoutValidation());
 
     thrown.expect(RuntimeException.class);
-    thrown.expectMessage("Failed to create load job");
+    thrown.expectMessage("Failed to create job");
```

Comment:
Can you add tests for some of the new utilities ? For example, PendingJobManager and PendingJob.

Reply:
done